README.md

```diff
@@ -308,13 +321,37 @@ Add the version of `solc` you want to use into `index.html`:
 ```html
 <script
   type="text/javascript"
-  src="https://binaries.soliditylang.org/bin/{{ SOLC VERSION }}.js"
+  src="https://binaries.soliditylang.org/bin/{{ SOLC_VERSION }}.js"
+  integrity="sha256-{{ BASE64_ENCODED_HASH_OF_SOLC_VERSION  }}"
+  crossorigin="anonymous"
 ></script>
 ```
 
-(Alternatively use `https://binaries.soliditylang.org/bin/soljson-latest.js` to get the latests version.)
+This will load `solc` into the global variable `window.Module`.
+Alternatively, use `https://binaries.soliditylang.org/bin/soljson-latest.js` to get the latest version.
```

Comment:
I thought about something like [latest version script](https://binaries.soliditylang.org/bin/soljson-latest.js), for example. Do you think this is appropriate?

Reply:
Oh right, my idea was to be more explicit about the location of the latest version since in the example above, it is shown as: `https://binaries.soliditylang.org/bin/{{ SOLC_VERSION }}.js`. And the link refers to an alternative to the example. But yes, we can omit the link if it makes it easier to understand.

Or maybe just: "Alternatively, replace `{{ SOLC_VERSION }}.js` by `soljson-latest.js` in the link above...`

Reply:
Understood. IMO, the latter suggestion is better, but just because I think that in general hiding the URLs behind an hyperlink is more elegant.

Reply:
But which latter suggestion? Yours or mine?
I think it should be clear that what we are referring to is the URL to be used in the script tag in the example. What about this one?

"Alternatively, use [soljson-latest.js](https://binaries.soliditylang.org/bin/soljson-latest.js) as `{{ SOLC_VERSION }}.js` in the code snippet above to load the latest version."

Reply:
I would suggest
```
Alternatively, use [soljson-latest.js](https://binaries.soliditylang.org/bin/soljson-latest.js) instead of {{ SOLC_VERSION }}.js in the code snippet above to load the latest version.
```